docs/source/start.ipynb

```diff
@@ -61,32 +53,40 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "automl = AutoMLSearch(problem_type=\"binary\", objective=\"f1\", max_batches=1)"
+    "X_train, X_holdout, y_train, y_holdout = evalml.preprocessing.split_data(X, y, problem_type='binary', test_size=.2)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In order to validate the results of the pipeline creation and optimization process, we will save some of our data as a holdout set."
+    "__Note:__ To provide data to EvalML, it is recommended that you create a `DataTable` object using [the Woodwork project](https://woodwork.alteryx.com/en/stable/). Here, `split_data()` returns Woodwork data structures.\n",
+    "\n",
+    "EvalML also accepts and works well with pandas `DataFrames`. But using the `DataTable` makes it easy to control how EvalML will treat each feature, as a numeric feature, a categorical feature, a text feature or other type of feature. Woodwork's `DataTable` includes features like inferring when a categorical feature should be treated as a text feature. For this reason, if you don't provide Woodwork objects, EvalML will raise a warning."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "X_train, X_holdout, y_train, y_holdout = evalml.preprocessing.split_data(X, y, test_size=.2)"
+    "EvalML has many options to configure the pipeline search. At the minimum, we need to define an objective function. For simplicity, we will use the F1 score in this example. However, the real power of EvalML is in using domain-specific [objective functions](user_guide/objectives.ipynb) or [building your own](user_guide/objectives.ipynb).\n",
+    "\n",
+    "Below EvalML utilizes Bayesian optimization (EvalML's default optimizer) to search and find the best pipeline defined by the given objective.\n",
+    "\n",
+    "EvalML provides a number of parameters to control the search process. `max_batches` is one of the parameters which controls the stopping criterion for the AutoML search. It indicates the maximum number of rounds of AutoML to evaluate, where each round may train and score a variable number of pipelines. In this example, `max_batches` is set to 1.\n",
```

Comment:
1. it might be better to move the explanation of `max_batches` until after the code example. it doesn't mean anything to see that parameter defined until it's actually used. for example, see in the featuretools docs how we explain the parameters to `entity_from_dataframe` at the end
![image](https://user-images.githubusercontent.com/371792/103246009-f5bc9700-492f-11eb-9b17-d3b127b6ac01.png)

2. not sure it's helpful to introduce the word "round" here since it's basically functioning as a synonym for batch without actually defining the word batch. 

3. We also don't give the user any intuition of when or why they'd want to use the parameter i.e it controls the "budget" evalml has to look for the best model. increasing the value allows evalml to search deeper for the best model, at the risk of overfitting.

Reply:
i see that this text wasn't actually change in the PR, so maybe best done as a follow up